tools/extra/fpgadiag/eth_group.cpp

```diff
@@ -54,140 +54,49 @@
 
 
 // open eth group
-int eth_group::eth_group_open(int vfio_id, std::string fpga_mdev_str)
+int eth_group::eth_group_open(std::string fpga_uid_str)
```

Comment:
Can fpga_uid_str be a const std::string&?

Reply:
Changed input to open to  const std::string&, fixed